compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/types/BLangTupleTypeNode.java

```diff
@@ -37,14 +38,21 @@
 public class BLangTupleTypeNode extends BLangType implements TupleTypeNode {
 
     // BLangNodes
-    public List<BLangType> memberTypeNodes = new ArrayList<>();
+    public List<BLangSimpleVariable> memberTypeNodes = new ArrayList<>();
```

Comment:
```suggestion
    public List<BLangSimpleVariable> members = new ArrayList<>();
```

Please check all fields and update appropriately.